File: 1 - BackEnd/Vanilla/PHP/3 - lang/2 - tipos_de_datos.php
<?php

/* ##########====================########## */
/* ######===--- Tipos de datos ---===###### */
/* ##########====================########## */

// Numeros.
$numero = 54353;
(float) $numero = 324235.435345;

// Cadena de texto.
$string = "Brandon Anthony";
$string = 'Brandon Anthony';

// Arreglo.
$array = [10, 20, 30];
$array = ["Hola", "Brandon", "Anthony"];
$array = [10, false, "Brandon"];
$array = array("Hola", "mundo", true);
$array = array(
	"nombre" => "Brandon", 
	"edad" => 22, 
	"Uno", 
	"arreglo" => ["Uno", "Dos", "Tres"]
);

// Booleanos.
(bool) $bool = true;
(bool) $bool = false;

// Sin valor, (nulo).
$nulo = null;

// Funciones anonimas.
$funcion_anonima = function():string{return "string";};
$funcion_anonima = fn ():string => return "string";

// Una clase anonima.
$clase_anonima = class{
	function __construct(){}
};

// Dato de tipo objeto.
$objeto = new Usuario();

// Dato estatico, (no modificable).
static $estatico = 5;

// Constantes.
define("PI", 3.1416);
define("Variable", "Este es el valor");
const PI = 3.1416;
const Variable = "Este es el valor";

File: 1 - BackEnd/Vanilla/Python/1 - lang/2 - tipos-de-datos.py
# ##########====================##########
# ######===--- Tipos de datos ---===######
# ##########====================##########

# Números.
numero = 54353
numero_float = 324235.435345

# Cadena de texto.
cadena = "Brandon Anthony"
cadena = 'Brandon Anthony'

# Listas.
lista = [10, 20, 30]
lista = ["Hola", "Brandon", "Anthony"]
lista = [10, False, "Brandon"]
lista = ["Hola", "mundo", True]
lista = {
    "nombre": "Brandon", 
    "edad": 22, 
    "Uno": None, 
    "arreglo": ["Uno", "Dos", "Tres"]
}

# Booleanos.
booleano = True
booleano = False

# Sin valor (None).
nulo = None

# Funciones anónimas (lambdas).
funcion_anonima = lambda: "string"

# Una clase.
class ClaseAnonima:
    def __init__(self):
        pass

# Dato de tipo objeto.
# Supongamos que tenemos una clase Usuario definida en otra parte.
class Usuario:
    pass

objeto = Usuario()

# Datos estáticos (en Python se usa el concepto de atributos de clase para similitudes).
class MiClase:
    estatico = 5

# Constantes.
PI = 3.1416
VARIABLE = "Este es el valor"